CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# context-template ChangeLog
+# ed25519-signature-2018-context ChangeLog
```

Comment:
```suggestion
# cryptosuite-context-ed25519-2018 ChangeLog
```

I'm wondering if we want to start imposing a pattern here... cryptosuite-context-SYSTEM-YEAR ... it defines more than just the signature mechanism, but public key format as well

Reply:
Consensus after today's conversation: we'll name these `<suitename>-<other pattern>`, so, in this case, since the suite name is 'Ed25519 Signature 2018' suite', we'll keep it as `ed25519-signature-2018-context`.